portal-api/src/v1/routes.js

```diff
@@ -205,4 +207,6 @@ authRouter.get('/validate', (req, res) => {
 // bank-validator
 authRouter.get('/validate/bank', (req, res) => banks.validateBank(req, res));
 
+authRouter.route('/previous-reports/:bankId').get(validateUserHasAtLeastOneAllowedRole({ allowedRoles: [PAYMENT_OFFICER] }), getPreviousReportsByBankId);
```

Comment:
It's missing  from the rest of this file like in the other route but we should validate bankId here with the validator package just as in central API. I think TFM API has an example of how we can write some validations there